### PR TITLE
Allow a "*" datacenter to indicate replication to everywhere

### PR DESCRIPTION
**Pull request by jypma**
Which will also automatically pick up new datacenters as they are added
(after a restart, of course)

### CODE REVIEW

File: ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/DataCenterForwarder.java [hunk 1]
```diff
@@ -216,7 +216,7 @@ private void updateLastEventOffset() {
                     return visibilityRepo.setMaster(e.persistenceId(), weAreMaster).thenApply(done -> Tuple.of(e, shouldMakeVisible));
                 } else {
                     return visibilityRepo.getVisibility(e.persistenceId()).thenApply(v -> {
-                        log.debug("visibility of {} is {}/{}", e, v.isMaster(), v.getDatacenters());
+                        log.debug("visibility of {} is {}/{}", e, v);
```
Comment (alar17) [inline ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/DataCenterForwarder.java]: Are there three parameters and two inputs?

File: ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/Visibility.java [hunk 2]
```diff
@@ -19,23 +19,20 @@
...
-        return datacenters;
-    }
-    
     public boolean isMaster() {
         return master;
     }
 
     public boolean isVisibleTo(DataCenter dataCenter) {
-        return datacenters.contains(dataCenter.getName());
+        return datacenters.contains("*") || datacenters.contains(dataCenter.getName());
```
Comment (alar17) [inline ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/Visibility.java]: Consider adding a comment explaining `*` here as well. It's not trivial to understand it at the first sight.

File: ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/DataCenterForwarder.java (hunk 1 above)
Comment (jypma) [reply ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/DataCenterForwarder.java]: good catch.

File: ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/Visibility.java (hunk 2 above)
Comment (jypma) [reply ts-reaktive-replication/src/main/java/com/tradeshift/reaktive/replication/Visibility.java]: yupp, adding.

**Pull request merged**